src/lib/autoinstall/y2erb.rb

```diff
@@ -12,7 +12,12 @@ def self.render(path)
 
     class TemplateEnvironment
       include Yast::Logger
-      include Y2Autoinstallation::CommonHelpers
+
+      # @see {EFIDetector}
+      # @return [Boolean] whether the system is booted using EFI or not
+      def boot_efi?
+        (@efi_detector ||= EFIDetector.new).boot_efi?
```

Comment:
do we really need to cache it when class holds nothing?

Reply:
No, it is not needed.